oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -32,6 +33,15 @@
   private static final URI CALL_URI = URI.create("http://googleapis.com/testapi/v1/foo");
 
   @Test
+  public void constructor_accessAndRefreshTokenNull_throws() {
+    try {
+      new UserCredentials(CLIENT_ID, CLIENT_SECRET, null, null);
+      fail("Should not be able to refresh without refresh token.");
+    } catch (Exception expected) {
+    }
+  }
```

Comment:
Do we not have access to the version of junit that supports @Test(expected=Exception) ?